src/js/components/Video.js

```diff
@@ -26,10 +28,11 @@ export default class Video extends Component {
     this._onClickChapter = this._onClickChapter.bind(this);
     this._onFullScreen = this._onFullScreen.bind(this);
 
-    this.state = { playing: false, progress: 0 };
+    this.state = { playing: false, progress: 0, responsive: false };
```

Comment:
Minor feedback.

Our pattern has been not to use generic names for the responsive option.

In your case you want a configurable icon size.

How about:


this.state = { playing: false, progress: 0, iconSize: 'large' };

and for the responsive option you can set the size to small:

```
 _onResponsive (small) {
    this.setState({iconSize: small ? 'small' : 'large' });
```